juju/juju
# [JUJU-4048] Remove GetChanges & series from GetChangesMapArgs

Remove GUIArgs and GetChanges bundle facade endpoints

GetChanges has been deprecated in favour of GetChangesMapArgs, which
doesn't rely clients being aware of a magical ordering. Remove
GetChanges now that we have a major version bump

This means we can remove GUIArgs alongside which is now dead code.

Also remove series from GetChangesMapArgs

As a side, drop both of these functions from these from `api/` since they aren't used in our client

Bump the facade version of bundles

## Checklist


## QA steps

The juju client doesn't use these endpoints, and our other clients (pylibjuju) don't support 4.0 yet, so all we can really do is ensure juju compiles and unit tests pass

Feel free to suggest something if I haven't thought of something
